lisp/ess-tracebug.el

```diff
@@ -1262,6 +1262,10 @@ If in debugging state, mirrors the output into *ess.dbg* buffer."
       ;; FIXME: this should be in comint filters!!
       ;; insert \n after the prompt when necessary
       (setq string (replace-regexp-in-string prompt-replace-regexp " \n" string nil nil 1))
+      ;; Replace + that gets printed twice in the output:
+      (when (or (not ess-eval-visibly)
+                (eq ess-eval-visibly 'nowait))
```

Comment:
this should be intergrated within the following `(when infrior-ess-replace-long+ ...`.

I think a better idea would be to multiple options for `inferior-ess-replace-long+` (nil, t, strip).